packages/amplify-provider-awscloudformation/src/utils/admin-helpers.ts

```diff
@@ -93,7 +93,9 @@ async function getAdminCognitoCredentials(idToken: CognitoIdToken, identityId: s
 }
 
 async function getAdminStsCredentials(idToken: CognitoIdToken, region: string): Promise<AwsSdkConfig> {
-  const sts = new aws.STS();
+  const sts = new aws.STS({
+    stsRegionalEndpoints: 'regional',
+  });
```

Comment:
is sts available in all regions ?

Reply:
Looks like there are a handful of regions it's not available in, but it's in all regions that studio supports
https://docs.aws.amazon.com/general/latest/gr/sts.html

Reply:
Is that codepath running only in studio ? If yes then we're good. If not then would sdk fall back to global? or would we need a setting or fallback code ?

Reply:
Yes this codepath only runs for studio enabled apps